src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: all ease 3s;
+  &:hover {
+    transform: scale(1.2);
+  }
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    padding-left: 3px;
+    padding-right: 3px;
```

Comment:
Adding padding to the left and right of the image may distort it. It's better to center the image using other techniques, for example, by setting the parent's display property to 'flex' and using 'justify-content: center;'